experiments/compositions/composition/api/v1alpha1/expanderversion_types.go

```diff
@@ -29,10 +29,17 @@ type ExpanderVersionSpec struct {
 	// Important: Run "make" to regenerate code after modifying this file
 
 	// ImageRegistry is the designated registry for where to pull the named expander image
-	ImageRegistry string `json:"imageRegistry"`
+	ImageRegistry string `json:"imageRegistry,omitempty"`
```

Comment:
Currently we are getting the image registry from the expanderVersion CR, so it would be required, why are we making it optional?

Reply:
This is not needed when type is `grpc` for now.

Reply:
I see, thank you!